src/chat.rs

```diff
@@ -2908,7 +2908,11 @@ async fn prepare_send_msg(
         CantSendReason::ProtectionBroken | CantSendReason::ContactRequest => {
```

Comment:
Maybe split this into separate branches to not mix different cases? `ProtectionBroken` isn't related to leaving groups/broadcasts here